system_health.py

```
@@ -0,0 +1,160 @@
+            cursor = conn.cursor()
+            cursor.execute('SELECT 1')
+            return True
+        except Exception as e:
+            logger.error(f"Database check failed: {e}")
+            return False
+
+    def check_internet(self):
+        try:
+            result = sandbox_command("ping", ["-c", "1", "8.8.8.8"])
+            return result.returncode == 0
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)